quic/s2n-quic/src/lib.rs

```diff
@@ -50,6 +50,10 @@
 //!
 //! **NOTE**: this will override the platform detection and always use [`s2n-tls`][s2n-tls] by default.
 //!
+//! ### `provider-fips-crypto`
+//!
+//! Enables use of FIPS-approved cryptography with s2n-quic.
+//!
```

Comment:
I plan on adding an API to easily enable fips security policy. I plan on expanding on this documentation at that point.